{"name":"InvalidLogin","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [false] but found [true]","trace":"java.lang.AssertionError: expected [false] but found [true]\r\n\tat org.testng.Assert.fail(Assert.java:96)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\r\n\tat org.testng.Assert.assertFalse(Assert.java:65)\r\n\tat org.testng.Assert.assertFalse(Assert.java:75)\r\n\tat com.qa.flipkart.test.TestRunner.InvalidLogin(TestRunner.java:47)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\r\n\tat org.testng.TestRunner.run(TestRunner.java:505)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\r\n\tat org.testng.TestNG.run(TestNG.java:1017)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)\r\n"},"stage":"finished","description":"This test is to check the login functionality with invalid details","steps":[{"name":"Verify Login using User phone 4433221100 and password 122aaaa","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"userphone","value":"4433221100"},{"name":"pass","value":"122aaaa"}],"start":1596727013721,"stop":1596727014649}],"attachments":[],"parameters":[],"start":1596727013518,"stop":1596727014994,"uuid":"d34eb185-b0ff-43d1-8967-f29c70d11a0a","historyId":"850698412bc90f95695a5b79ee3280d5","fullName":"com.qa.flipkart.test.TestRunner.InvalidLogin","labels":[{"name":"package","value":"com.qa.flipkart.test.TestRunner"},{"name":"testClass","value":"com.qa.flipkart.test.TestRunner"},{"name":"testMethod","value":"InvalidLogin"},{"name":"parentSuite","value":"Flipkart Automation Test Suite"},{"name":"suite","value":"Test"},{"name":"subSuite","value":"com.qa.flipkart.test.TestRunner"},{"name":"host","value":"Mypc"},{"name":"thread","value":"6840@Mypc.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"User Login"},{"name":"severity","value":"blocker"}],"links":[]}